apps/system/test/marionette/lib/system.js

```diff
@@ -389,6 +389,17 @@ System.prototype = {
     return frame;
   },
 
+  closeBrowser: function(url) {
```

Comment:
I'd like to retain the word `browser` when we interact with web pages (See `system.gotoBrowser(url)`). I'll call it `closeBrowserByUrl` if that OK with you.